astrosat_users/serializers/serializers_auth.py

```diff
@@ -209,6 +209,7 @@ def validate(self, data):
             )  # little hack here b/c I'm using a view outside a request
             emailconfirmation = view.get_object()
             data["confirmation"] = emailconfirmation
+            data["user"] = emailconfirmation.email_address.user
```

Comment:
Is this necessary, already adding the `emailconfirmation`, which has this. I understand if it is just more convenient though.

Reply:
As you say, it's more convenient.  And it makes it more intuitive to access it in `VerifyEmailView`.